Cloud.md

```diff
@@ -66,11 +57,190 @@ Relevant experience:
+- I'm not afraid to have difficult conversations when needed, and do so with tact and care.
+- I set clear expectations for people and they know how they are progressing against those expectations. I create bounded environments for action and let people have freedom within those boundaries.
+- I know when and how to apply different coaching and mentoring techniques, understanding the maturity level of those I'm helping.
+- I am able to provide perspective for those I'm serving by reminding them of the big picture when they get stuck in the details and day to day of solving problems.
+
+#### I know the pressures of being a key leader.
+
+- I know how to identify business problems/goals and evaluate the ROI and TCO of potential solutions.
+- I understand how budgeting, competition, strategic alignment and compliance pressures can influence technology decisions and am able to adjust proposed solutions based on those factors.
+- I understand the bigger picture of how software development, compliance, marketing and other business departments all interrelate and influence the success of a business.
+- I know how to adjust my communications style based on my audience, from support to engineer to sales and marketing to C-level executives and business owners.
+- I know the impact of internal politics in organisations and how it can alter decision making processes. I work to minimise any such activities and proactively display a culture of ownership and inclusion.
+- I know what is required to bring about organisational change, and when enough trust has been established to broach such topics with others.
+
+#### I am a servant leader for my team.
+
+- I am prioritising the needs of my team members above my own, knowing that their success is our success.
+- I am an outspoken advocate of my team. They can trust me to be the first line of defence.
+- I know the buck stops with me. At the end of the day I am responsible for my team and the relationship with our peers and customers.
+- I actively track the progress of our team, keeping the goals of our projects and our constraints front of mind. I make this information visible and transparent internally and externally to minimise 'surprises' and to facilitate early and pragmatic problem resolution.
+- I have a track record of delivering successful projects by applying good practices when leading teams.
+- I don't mind making the coffee or picking up the pizza for everyone in my team.
+- I am focussed on helping my team grow, sharing what I have learned and encouraging others to do the same.
+- I actively delegate tasks to my team providing them autonomy to complete the task. I trust my team to complete tasks and seek help when needed.
+- I understand culture is held in our people. I am proactive in hiring for good culture fit and attitude as much as skills.
+- My validation and success comes from the cumulative successes of my team.
```

Comment:
Feel free to ignore me but my preference would be to not have this statement. I think I get your sentiment but I fear others may read it differently to what you intended. I would hope the other examples would show they are recognised for how their team achieves rather than themselves.

Reply:
updated and removed.